test/cucumber/UpdateReport.feature

```diff
@@ -0,0 +1,11 @@
+Feature: UpdateReport
+  As a professor
+  I want to update report in the system
+  So I can save the new report
+    
+#Cenario Controller
+  Scenario: Update a report
+     Given I register "relatorio de teste de report" with tipo "Porcentagem" with valor "5" and avaliacao "MANA" 
```

Comment:
aqui teria que ser algo como: "the system has a report"...
Esse reuso não caberia aqui. Terminou precisando fazer dois passos quando só precisava de um.